assets/js/modules/analytics-4/components/common/WebDataStreamNameInput.test.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * WebDataStreamNameInput component tests.
```

Comment:
```suggestion
 * Analytics WebDataStreamNameInput component tests.
```

Maybe useful for easier searching/scanning

Reply:
Updated, thanks!